### DISCUSSION TITLE
Is it possible to turn off env variable expansion

### DISCUSSION THREAD

**Discussion by jamesshenry** [PowerShell/PowerShell#17774] [Q&A]
Is it possible to disable the expansion of Environment variables in the shell when using tab completion?

### Example
`$env:LocalAppData\Programs`
Press `<Tab>`
Becomes:
`C:\Users\<username>\AppData\Local\Programs\`

**Comment by masterflitzer** [comment]
i would love to have this, tab expanding env vars doesn't make any sense, i mean why am i using a variable? so i dont have to work with the long name
